mne/time_frequency/tests/test_spectrum.py

```diff
@@ -69,7 +70,26 @@ def test_compute_psd_errors(raw):
             True,
             False,
             "full",
+            "power",  # XXX: technically a default
         ],  # non-defaults
+        [
+            0,
+            np.inf,
+            None,
+            None,
+            None,
+            False,
+            256,
+            0,
+            None,
+            "mean",
+            "hamming",
+            None,
+            False,
+            True,
+            "length",
+            "complex",
+        ],  # complex  XXX: need to also test with non-defaults?
     ],
 )
 def test_spectrum_params(
```

Comment:
For testing I re-implemented a lot of the previous stuff with some new additions, e.g. here an extra test for `compute_psd()` with `output="complex"`.